include/openenclave/internal/trace.h

```diff
@@ -34,6 +34,7 @@ typedef void (*oe_log_callback_t)(
     const char* time,
     long int usecs,
     oe_log_level_t level,
+    uint64_t host_thread_id,
```

Comment:
Is this change supposed to be part of this PR? It doesn't look used anywhere.

Reply:
Yes this change is part of the oe_log_message() callbak feature.